[PCM-52] Add initial integration tests for Video Troubleshooting.

jbelmont opened the pull request:
I need to figure out why phantomjs can't seem to find dom elements for integration test. It passes in chrome but not in phantomjs

jensengar commented on package.json:
```diff
@@ -10,7 +10,8 @@
     "start": "ember server",
     "coverage": "COVERAGE=true ember test",
-    "test": "node_modules/.bin/semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
+    "test": "semistandard blueprints/**/*.js tests/**/*.js addon/**/*.js app/**/*.js && npm run coverage",
```
lets use local libraries, not global.

jbelmont replied:
this is using local libraries, it is unnecessary to to preface with node_modules/.bin it already does that

jensengar commented on tests/integration/components/webrtc-troubleshooter/component-test.js:
```diff
@@ -1,19 +1,94 @@
+    return this.t(...arguments);
+  },
+  t (key) {
+    if (key === 'chat.presence.outOfOffice') {
```
why do we care about this?

jbelmont replied:
this line is pointless only need to mockout t function so I will remove this condition

xdumaine commented on tests/dummy/translations/en-us.json:
```diff
@@ -0,0 +1,37 @@
+{
+  "webrtcTroubleshoot": {
```
Is there a way to have the dummy app load translations from the addon instead of duplicating strings?

jbelmont replied:
I haven't investigated it this like I should, but I can now

jbelmont commented:
@xdumaine I added the translation file to resolve errors in phantomjs but am not seeing them when running in headless chrome so I removed the duplication here. Although I am still looking at if there is some option at ember-cli-build.js that I can add to include intl assets
